projects/ngrx.io/content/marketing/presskit.md

```diff
@@ -0,0 +1,21 @@
+<h1 class="title center no-toc">Press Kit</h1><br />
+The logo graphics available for download on this page are provided under <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC BY 4.0</a>.
+
+## NgRx Logo
+<img src="assets/images/ngrx-badge.png" width="100" style="float:left; padding-right: 1rem;" />
+<div style="padding-top:1rem;">
+Logo (png) - <a href="#">Download</a><br />
+Logo (svg) - <a href="#">Download</a>
+</div>
+
+## NgRx Conference Logo
+<img src="generated/images/marketing/home/ngrx-conf-badge.svg" width="100" style="float:left; padding-right: 2rem;" />
+<div style="padding:1rem 0;">
+Logo (svg) - <a href="#">Download</a>
+</div>
+
+## State Management Lifecycle Diagram PSD
+
+<img src="generated/images/guide/store/state-management-lifecycle.png" width="100" style="float:left; padding-right: 1rem;" />
+Diagram (psd) - <a href="generated/images/guide/store/state-management-lifecycle.psd">Download</a><br />
```

Comment:
What about adding a download attribute?
It will download the file instead of opening it in the same window.

```suggestion
Diagram (psd) - <a href="generated/images/guide/store/state-management-lifecycle.psd" download>Download</a><br />
```